documentation/phylogenetic_queries.md

```diff
@@ -49,3 +49,28 @@ The result of such a query is a ndjson with a single row, where `missingFromTree
   "missingFromTree": "MISSING_NODE1,MISSING_NODE2"
 }
 ```
+
+### Subtree
+
+```json
+"action": {
+  "type": "Subtree",
```

Comment:
@Taepper and @chaoran-chen before I merge this is this a good name for this action or should I maybe rename it to sth more clear like `PhyloSubtree`?

Reply:
We discussed and decided on PhyloSubtree! :-)